redash/utils/__init__.py

```diff
@@ -82,7 +82,7 @@ def default(self, o):
         elif isinstance(o, (datetime.timedelta, uuid.UUID)):
             result = str(o)
         # See "Date Time String Format" in the ECMA-262 specification.
-        if isinstance(o, datetime.datetime):
+        elif isinstance(o, datetime.datetime):
```

Comment:
@jezdez I assume this was accidental, right?

Reply:
Looks like it, yeah!